mutation/mutation_partition_v2.cc

```diff
@@ -110,9 +110,13 @@ void mutation_partition_v2::ensure_last_dummy(const schema& s) {
     }
 }
 
-std::ostream& operator<<(std::ostream& out, const apply_resume& res) {
-    return out << "{" << int(res._stage) << ", " << res._pos << "}";
-}
+template <>
+struct fmt::formatter<apply_resume> : fmt::formatter<std::string_view> {
```

Comment:
What does inheriting from an existing formatter provides?

Reply:
good question. it provides

1. the format specifier. and much less importantly, 
2. we can just call the parent's `formatter<std::string_view>::format("foobar", ctx)` when appropriate,
3. and less typings. 

regarding the first point, fmtlib provides the the implementation of `fmt::formatter<T>`, which offers its own format specifiers. where `T` is its core types, like bool, float, int, and `string_type`. and `basic_string_view<char>` is considered as `string_type`. so we will have the format specifier support automatically. despite that we don't use the format specifier widely in our project, but i think once we are familiar with it, it would be very handy.